kustomize/README.md

```diff
@@ -0,0 +1,228 @@
+## INSTALLING TERRASCAN IN A KUBERNETES CLUSTER USING KUSTOMIZE
+
+This guide will help you install terrascan server inside your kubernetes cluster, as per your specific use case.
```

Comment:
I believe, we can remove the heading and add bullets points for each section with hyperlinks pointing to that specific section